manifests/latest/vsphere-7.0u2/supervisorcluster/k8s-1.17/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -0,0 +1,185 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: vsphere-csi-controller
+  namespace: vmware-system-csi
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: vsphere-csi-controller
+  template:
+    metadata:
+      labels:
+        app: vsphere-csi-controller
+        role: vsphere-csi
+    spec:
+      serviceAccount: vsphere-csi-controller
+      nodeSelector:
+        node-role.kubernetes.io/master: ''
+      tolerations:
+        - operator: "Exists"
+          key: "node-role.kubernetes.io/master"
+          effect: "NoSchedule"
+        - operator: "Equal"
+          key: "kubeadmNode"
+          effect: "NoSchedule"
+          value: "master"
+      hostNetwork: true
+      containers:
+        - name: csi-provisioner
+          image: vmware/csi-provisioner/csi-provisioner:v1.2.1_vmware.2
+          args:
+            - "--v=4"
+            - "--timeout=300s"
+            - "--csi-address=$(ADDRESS)"
+            - "--feature-gates=Topology=true"
```

Comment:
We do not support topology in WCP. Can you remove these flags?
Refer - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/381

Reply:
Yes just saw your PR, changing them